aliases.sh

```diff
@@ -0,0 +1,9 @@
+alias git-graph='git log --all --graph --date-order --oneline --decorate';
+alias git-current-branch='git rev-parse --abbrev-ref HEAD';
+alias git-current-commit='git rev-parse --verify --short HEAD';
+alias git-current-tag='git fetch --tags --force --quiet && git describe --tags 2> /dev/null';
+alias git-reset='git reset --hard origin/$(git-current-branch)';
+alias git-clean-local-branches='git fetch --quiet && git branch -vv | grep -v  "^\*" | grep "\[.*: gone\]" | cut -d " " -f 3 | xargs git branch --delete';
+
+alias package-version='function _composer_package_version { parts=($(echo ${1%/} | tr "/" "\n")); if [[ 2 -le $#parts ]]; then vendor="${parts[-2]}/"; else vendor=''; fi; package="$vendor${parts[-1]}"; grep "\"name\": \".*$package\"" -A 5 composer.lock; }; _composer_package_version';
+alias composer-ulm='php -d memory_limit=-1 $(where composer)';
```

Comment:
On Linux, it would be `alias composer-ulm='php -d memory_limit=-1 $(whereis -b composer | cut -f 2 -d : | sed "s/ *//g")';`.

Reply:
`which` is available on both macOS and Linux